packages/cli/src/cmds/account.ts

```diff
@@ -169,14 +181,37 @@ class AccountLogin extends YargsCommand {
         description: "Authenticate with a specific Azure Active Directory tenant.",
         type: "string",
         default: "",
+      })
+      .options("service-principal", {
```

Comment:
Is this to follow the same behavior as az-cli? Add error handlings for all unsupported cases:
- -u xxx@xxx.com -p SECRET (user login)
- --service-principal -u CLIENTID -p CERTIFICATE -t TENANTID (cert credential)
- --service-principal -u CLIENTID -p SECRET (no tenant)

Call out in documentation/example that the only supported case is **--service-principal -u CLIENTID -p SECRET -t TENANTID**

Reply:
Already add parameter check logic. And add examples to command help.
![image](https://user-images.githubusercontent.com/13211513/145140885-c1e60b32-6e94-4d87-a6b9-984da62904a7.png)